drape_frontend/animation/animation.cpp

```diff
@@ -90,4 +90,41 @@ bool Animation::GetMaxDuration(Interpolator const & interpolator, double & maxDu
   return true;
 }
 
+std::string DebugPrint(Animation::Type const & type)
+{
+  switch (type)
+  {
+  case Animation::Type::Sequence: return "Sequence";
+  case Animation::Type::Parallel: return "Parallel";
+  case Animation::Type::MapLinear: return "MapLinear";
+  case Animation::Type::MapScale: return "MapScale";
+  case Animation::Type::MapFollow: return "MapFollow";
+  case Animation::Type::Arrow: return "Arrow";
+  case Animation::Type::KineticScroll: return "KineticScroll";
+  default: return "Unknown type";
```

Comment:
Please, don't use default unless you're really want to handle all possible cases. Just
```return "Unknown type";``` at the end of the function.